samplesheets/views.py

```diff
@@ -2538,37 +2561,42 @@ class IrodsRequestAcceptView(
     IrodsRequestModifyMixin,
     FormView,
 ):
-    """View for accepting an iRODS data request"""
+    """View for accepting iRODS data requests"""
```

Comment:
One major bug: when I click on "Accept Request" on a request where the path is a collection, I get the following error: `'dict' object has no attribute 'path'`

Please fix and also introcude test(s) which ensure the form renders and works properly on requests which target collections instead of data objects.